test/unit/client/test_build_runner.py

```diff
@@ -7,36 +7,87 @@
 
 class TestBuildRunner(BaseUnitTestCase):
 
-    def mock_runner_with_status_response(self, response):
+    def _create_build_runner_instance(self, response):
+        mock_get_logger = self.patch('app.client.build_runner.get_logger')
+        logger = MagicMock()
+        mock_get_logger.return_value = logger
```

Comment:
I was curious about a very slight potential simplification, replacing lines 12 and 13 with:
```python
logger = mock_get_logger.return_value
```
I think accessing the return_value directly should return a Mock -- and depending on how it's used, you may or may not to replace it explicitly with a MagicMock. I think it's fine either way -- it was just something that I was thinking about. :wink: